app/scripts/modules/ecs/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -609,6 +661,7 @@ export class EcsServerGroupConfigurationService {
         this.configureAvailableSecrets(command);
         this.configureAvailableServiceDiscoveryRegistries(command);
         this.configureAvailableRegions(command);
+        this.setCapacityProviderDetails(command);
```

Comment:
Does this actually work?

Reply:
Yes, credentialsChanged does get invoked when we select the account in SG wizard